api2-war/src/main/java/eu/europeana/api2/v2/web/controller/ExceptionControllerAdvice.java

```diff
@@ -0,0 +1,49 @@
+/*
```

Comment:
Definitely agree.  Refactored all tag related code in eu.europeana.corelib.search.utils.FakeTagsUtils and now the controller refers to these helpers directly. reduced the number of lines by 30%